### PR TITLE
Add mapOf method for Tuple2

### LINKED ISSUE #134: Add mapOf for Tuple2
Create a mapOf function to build a Map from Tuple2 as the mapOf (in kotlin-stdlib) builds a Map from Pair

### PR DESCRIPTION
**Pull request by npatarino**
I don't know where is the best package to locate this method.
I've put it in a similar place than kotlin-stdlib put

This PR wants to resolve #134

### CODE REVIEW

File: kategory/src/main/kotlin/kategory/collections/Maps.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package kategory.collections
+
+import kategory.Tuple2
+
+fun <K, V> mapOf(vararg tuple: Tuple2<K, V>): Map<K, V> =
```
Comment (pakoito) [inline kategory/src/main/kotlin/kategory/collections/Maps.kt]: NIT: tuples as it's an array

Comment (raulraja) [inline kategory/src/main/kotlin/kategory/collections/Maps.kt]: The package should be just `package kategory`. I'm gonna approve it but please get it fixed before we merge it.

Comment (npatarino) [inline kategory/src/main/kotlin/kategory/collections/Maps.kt]: Done ;)

**Comment by raulraja** [comment]
@npatarino Thanks for your contribution!

**Comment by npatarino** [comment]
Thanks to you all!
❤️

**Pull request merged**